client/src/models/footnotes/dynamic_footnotes.yaml

```diff
@@ -29,15 +29,15 @@ late_results_warning_crso:
   fr: Les données du {{result_doc_name}} de cette responsabilité essentielle ne sont pas encore disponibles. Une mise à jour suivra.
 
 
-late_planned_spending_warning_gov:
-  en: Planned spending data does not include values from the organizations listed below, as their data is not yet available. Updates will follow.
+late_resources_warning_gov:
+  en: Planned spending and planned FTE data does not include values from the organizations listed below, as their data is not yet available. Updates will follow.
```

Comment:
Technically should probably be and/or, right? But I don't think we should change it.

Reply:
It's possible someone fills out one and not the other, but I think we expect it to be all or nothing?

Reply:
I mean, hopefully...